filebeat/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# 0.1.1
+[giladsh]
+- fixed registrty file parsing error - the file contains a list of dictionaries, not a dictionary
```

Comment:
Seems that changed with ELK v5.x, registry files were of the form https://github.com/DataDog/integrations-extras/pull/48/files#diff-9e185e1d6504fce30e41ef63f6bf4c18 before that.

So two things here:
* we probably want to maintain backward-compatibility with filebeat < 5.x format
* unit tests should test both formats